textproc/chordii/Portfile

```diff
@@ -0,0 +1,52 @@
+PortSystem          1.0
+
+name                chordii
+version             4.5.1
+revision            0
```

Comment:
It just doesn't make sense to have it there, but it is not really a problem. (Pure cosmetics.)